package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quintype/seo",
-  "version": "1.36.0",
+  "version": "1.34.2",
```

Comment:
here you are loosing 1.36.0 and 1.35.0, 1.35.1 changes.

Reply:
@shraddha-kesari Talk to @iamgarima on how to publish this as master is a protected branch

Reply:
Okay. Will talk to her.